R/opq.R

```diff
@@ -241,59 +241,133 @@ add_osm_feature <- function (opq,
                              value_exact = TRUE,
                              match_case = TRUE,
                              bbox = NULL) {
+  if (missing(key)) {
```

Comment:
These lines, and a lot of others, are flagged as having been changed, because you've changed in the indentation from 4 spaces to 2. An easy way to ensure you've got the same style including spacing is to run:
``` r
styler::style_file ("R/opq.R", style = spaceout::spaceout_style)
```
where [`spaceout` is here](https://github.com/ropensci-review-tools/spaceout). That should revert all spacing etc. Thanks!

Reply:
Thanks! I should have asked what styler to use – I wasn't familiar with the spaceout_style and kept trying to correct manually.